micromamba/tests/test_env.py

```diff
@@ -307,6 +307,7 @@ def test_env_update_conda_forge_with_pypi(tmp_home, tmp_root_prefix, tmp_path):
 channels:
 - conda-forge
 dependencies:
+- python 3.10
```

Comment:
This is to fix the CI test failure due to some known issues with `python 3.11` and `pip`.